### ISSUE TITLE
Locals spawning at my house

### ISSUE THREAD

**Issue by NickNubbz** [ParadoxGamingRP/bugs-and-suggestions#1539]
**Your Discord Name**
NickNubbz

**Describe the annoyance**
This in no way is a major issue but is pretty annoying when there are 10 locals in my front yard and using my pool in the back and also cars spawning in the drive way. i honestly think any player owned property should be local free.

**To Reproduce**
[Steps to reproduce the annoyance]
1. own a house
2. 
3. 

**Requested behavior**
possibly make all player owned property local free including their cars in the driveway.

**Screenshots**
did not get screenshot

**Additional context**
[Add any other context about the enhancement here]

**Comment by ithegreatgatsby** [comment]
If we did this for a few people we would have to do it to all housing and then we would get into things being messed up with the script checking for so many places to despawn peds. So at this time its not something we can really do.

**Comment by NickNubbz** [comment]
No worries. Just something that came to mind I figured I would suggest. Thank you for the response!Sent from my iPhoneOn Jun 23, 2023, at 3:02 PM, Chris ***@***.***> wrote:﻿
If we did this for a few people we would have to do it to all housing and then we would get into things being messed up with the script checking for so many places to despawn peds. So at this time its not something we can really do.
